packages/components/psammead-media-player/src/Placeholder/index.jsx

```diff
@@ -12,15 +14,57 @@ const StyledPlaceholder = styled.div`
   height: 100%;
 `;
 
-const Placeholder = ({ onClick, src }) => (
-  <StyledPlaceholder onClick={onClick}>
-    <Image alt="Image Alt" src={src} />
-  </StyledPlaceholder>
-);
+const StyledPlayButton = styled(PlayButton)`
+  position: absolute;
+  bottom: 0;
+
+  /* stylelint-disable */
+  /* https://www.styled-components.com/docs/advanced#referring-to-other-components */
+  ${StyledPlaceholder}:hover &,
+  ${StyledPlaceholder}:focus & {
+    background-color: ${C_POSTBOX};
+  }
+  /* stylelint-enable */
+`;
+
+const Placeholder = ({ onClick, service, src, mediaInfo }) => {
+  const { title, datetime, duration, durationSpoken, type } = mediaInfo;
+  return (
+    <StyledPlaceholder onClick={onClick}>
+      <StyledPlayButton
+        title={title}
+        service={service}
+        onClick={() => {}}
+        datetime={datetime}
+        duration={duration}
+        durationSpoken={durationSpoken}
+        type={type}
+      />
+      <Image alt="Image Alt" src={src} />
+    </StyledPlaceholder>
+  );
+};
 
 Placeholder.propTypes = {
   onClick: func.isRequired,
+  service: string.isRequired,
   src: string.isRequired,
+  mediaInfo: shape({
+    title: string.isRequired,
+    datetime: string,
+    duration: string,
+    durationSpoken: string,
+    type: 'video',
```

Comment:
I think you copied the default prop value here by mistake. This should be a prop type – `oneOf(['audio', 'video'])`.